src/trading/postCoWProtocolTrade.ts

```diff
@@ -69,5 +61,5 @@ export async function postCoWProtocolTrade(
 
   log(`Order created, id: ${orderId}`)
 
-  return orderId
+  return { orderId, signature, signingScheme }
```

Comment:
`OrderSigningUtils` might change (fallback) `signingScheme`, so we should return those values in order to know which `signingScheme` was actually used